src/pkg/build/types/definition_parser_deffile_test.go

```diff
@@ -135,7 +136,8 @@ func TestParseDefinitionFileFailure(t *testing.T) {
 			}
 			defer defFile.Close()
 
-			if _, err = ParseDefinitionFile(defFile); err == nil {
+			if d, err := ParseDefinitionFile(defFile); err == nil {
+				fmt.Println(d)
```

Comment:
Purpose of `fmt.Println()` here? Forgot to remove a debug statement? We should rather be using a `t.Log` statements to log here

Reply:
missed debug statement